src/controllers/tvhome/home.js

```diff
@@ -5,7 +5,7 @@ define(['connectionManager', 'loading', 'globalize', 'tabbedPage', 'libraryMenu'
 
         var homeScrollContent = page.querySelector('.contentScrollSlider');
 
-        html = html;
+        self.html = html;
```

Comment:
Why?

Reply:
i think i should remove this line ` self.html = html;`?

Reply:
`self.html` isn't used anywhere in the file, so I'm struggling to understand what you were trying to do :)

Reply:
i mean like this https://github.com/jellyfin/jellyfin-web/pull/810/commits/57e4287c8301be8335de4f1c9966ca093736a8f9